esp/services/ws_logaccess/WsLogAccessService.cpp

```diff
@@ -11,37 +11,17 @@ Cws_logaccessEx::~Cws_logaccessEx()
 bool Cws_logaccessEx::onGetLogAccessInfo(IEspContext &context, IEspGetLogAccessInfoRequest &req, IEspGetLogAccessInfoResponse &resp)
 {
     bool success = true;
-    if (m_remoteLogAccessor)
+    if (queryRemoteLogAccessor())
```

Comment:
future: fine as it is now, but when or if an auto-updating mechanism is added (HPCC-27983), this will method will likely want to turn into getRemoteLogAccessor, so that it links the object, and store it in a temporary Owned, for use below. That way the update can change the object, whilst the existing object is still valid.